10LessonTen/Checkpoint2/index.html

```diff
@@ -1,10 +1,117 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <title>Lesson Ten | Check Point</title>
+    <link rel="stylesheet" href="css/style.css">
+
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700|Lora&subset=latin,greek' rel='stylesheet' type='text/css'>
 </head>
+
 <body>
 
+    <!--nav bar starts-->
+    <div id="navigation">
+        <div id="navtitle">THE
+            <STRONG>KANGAROO</STRONG> EXPERIENCE</div>
+        <ul>
+            <li><a href="#">CONTACT</a></li>
+            <li><a href="#">SAFETY</a></li>
+            <li><a href="#">ABOUT</a></li>
+            <li><a href="#">HOME</a></li>
+        </ul>
+    </div>
+    <!--nav bar ends-->
+
+
+    <!--title card starts-->
+    <div id="titlecard">
+        <div id="titletext">
+            <h1>THE <strong>KANGAROO</strong> EXPERIENCE</h1>
+            <h2>SAFE ENCOUNTERS WITH AUSTRALIA'S GENTLE HORSE</h2>
+        </div>
+        <div class="button"><a href ="#">GO OUTBACK</a></div>
+    </div>
+    <!--title card ends-->
+
+
+    <!--section one starts-->
+    <section id="section1">
+        <div class="section1half" id="left">
+          <img src="img/kangaroo2.jpg" />
+        </div>
+        <div class="section1half" id="right">
+            <h2>BECOME <strong>MATES</strong> WITH THIS SAFE ANIMAL</h2>
+            <h6>'Roos get a bit of a bad reputation for "kicking tourists" and being somewhat "punchy." What a load of wallaby waste, mate! Here at The Kangaroo Experience, we aim to clean up the perception of these energetic cuddle hoppers. (We'd never have put our life savings into starting this company if it turned out they were dangerous, heh!) We expect that most or even all of them will refrain from pummeling you and your expedition partners on this certainly safe hour-long encounter! </h6>
+        </div>
+    </section>
+    <!--section one ends-->
+
```

Comment:
A. I love this page. B. Kangaroos are sweet. C. This page makes me happy. D. I love the s.a.f.e. letters in that section! D. your kanga4 picture needs to be linked EXACTLY as the extension says (for this one, it's .JPG not .jpg for some reason).

Great work!!